x-pack/plugins/infra/public/components/infrastructure_node_metrics_tables/pod/use_pod_metrics_table.ts

```diff
@@ -33,13 +33,14 @@ const podMetricsMap: MetricsMap<PodMetricsField> = {
   },
 };
 
-const { options: podMetricsOptions, metricByField } = metricsToApiOptions(
-  podMetricsMap,
-  'kubernetes.pod.name'
-);
+const { options: podMetricsOptions, metricByField } = metricsToApiOptions(podMetricsMap, [
+  'kubernetes.pod.uid',
+  'kubernetes.pod.name',
```

Comment:
I'm still not sure if I like this approach. It adds to the potential number of buckets and some what dilutes the meaning of the `groupBy` field since we're only asking for this to use it in the URLs, not to actually group by it (though the final effect should be the same since the fields both identify pods).

The problem is that we don't have an alternative. Because both `name` and `uid` are mapped as keywords and the Metrics Explorer API doesn't support anything but metric aggregations (min, max, avg, etc.). 

This is something we see in other places too, like start_time (which thank fully is typed as `date`), where we ask for the time series values but only use the max value. 
If we ever rewrite the API that backs this component, these are things we should look to fix since it's really not clear nor convenient in the code.

Reply:
I understand your concern. I'm assuming here that `uid` and `name` will correlate to the same set of docs. That said, the group by will keep its current behaviour. We could have however duplicate items if the same name is attributed to a different `uid`, but fortunately, the name in k8s land is unique